geomesa-tools/conf/log4j.properties

```diff
@@ -4,7 +4,7 @@ log4j.rootLogger=info,file
 geomesa.log.dir=.
 
 # set logging levels to appropriate values
-log4j.logger.org.locationtech.geomesa.tools=info,stderr
+log4j.logger.org.locationtech.geomesa.tools=info
```

Comment:
hahaha i bet this will gitblame to either me or @elahrvivaz that's funny that its the fix